core/controllers/question_editor.py

```diff
@@ -30,11 +30,19 @@ class QuestionCreationHandler(base.BaseHandler):
     """A handler that creates the question model given a question dict."""
 
     @acl_decorators.can_manage_question_skill_status
-    def post(self, skill_id):
+    def post(self, skill_ids):
         """Handles POST requests."""
-        skill_domain.Skill.require_valid_skill_id(skill_id)
-        skill = skill_services.get_skill_by_id(skill_id, strict=False)
-        if skill is None:
+        skill_ids_list = skill_ids.split(',')
+        if len(skill_ids_list) > 3:
+            raise Exception(
+                'More than three QuestionSkillLink for one '
+                'question is not supported.')
+
+        for skill_id in skill_ids_list:
+            skill_domain.Skill.require_valid_skill_id(skill_id)
+        try:
+            skill_services.get_multi_skills(skill_ids_list)
+        except:
             raise self.PageNotFoundException(
                 'The skill with the given id doesn\'t exist.')
```

Comment:
which ID? You're fetching multiple skills here

Reply:
Done.